ComponentKit/Utilities/CKComponentAction.mm

```diff
@@ -76,8 +76,12 @@ void CKConfigureInvocationWithArguments(NSInvocation *invocation, NSInteger inde
   id responder = ([target respondsToSelector:@selector(targetForAction:withSender:)]
                   ? [target targetForAction:selector withSender:target]
                   : target);
-  CKCAssertNotNil(responder, @"Unhandled component action %@ following responder chain %@",
-                  NSStringFromSelector(selector), _CKComponentResponderChainDebugResponderChain(target));
+
+  // If we have no responder to start, bail early.
+  if (responder == nil) {
```

Comment:
How are you differentiating between a nil responder, and a weak responder that has been deallocated? A straight-up nil target I think is fine to remove the assert. However, targets are held weakly, and a common error is them being deallocated before the action fires, which would now not be caught at all, right?